app/views/merchant/invoices/show.html.erb

```diff
@@ -22,5 +22,7 @@
   </div>
 <% end %>
 
-
-
+<div id="total-revenue">
```

Comment:
This displays a bit confusingly, but I'll update this as I'm refactoring everyhing.